src/omnicore/utils.cpp

```diff
@@ -28,9 +28,10 @@
  * https://github.com/mastercoin-MSC/spec#class-b-transactions-also-known-as-the-multisig-method
  *
  * @param strSeed[in]      A seed used for the obfuscation
+ * @param hashCount[in]    How many hashes to generate (number of packets to debofuscate)
```

Comment:
Looks like you swapped the order of the parameter (vstrHashes not before hashCount).